app/components/bali/progress/index.scss

```
@@ -0,0 +1,23 @@
+$progress-value-bar-color: var(--progress-value-bar-color);
+$progress-background-bar-color: lightgray;
```

$grey-light o su equivalente

```suggestion
$progress-background-bar-color: $grey-light;
```